dashboard/app/views/home/_privacy.md

```diff
@@ -122,13 +122,11 @@ A pixel tag (also known as a “clear GIF” or “web beacon”) is a tiny imag
-
-Certain capabilities linked from the Code.org Website may be implemented by third parties who collect data. For example, donations to Code.org are processed via a third party who collects and may store the payment instrument. Support messages are processed by a third-party site which may allow Users to create profiles and post messages. Other third-party websites, such as the Code.org [Facebook page](https://www.facebook.com/Code.org) and [Twitter page](https://twitter.com/codeorg), may be co-branded with our name or logo. Use of these third-party services are completely optional to Users, typically meant only for adults, and governed by the privacy policy of these third parties. We encourage you to review the privacy policy at the third parties for details about how they may collect, store, and use your Personal Information.
+Code.org may use a variety of third-party service providers to support our operations.  For example, we may use third parties such as email service platforms to send email, analytics companies to understand our Website usage and performance, and social networking platforms to host our videos. Similarly, we may use third-party service providers to implement and host our Website and associated services and features, provide teacher forums, process donations, provide contract and workflow management services, assist with customer support, and provide other functions in support of our organization.  When we give service providers access to data, including Personal Information, they are only allowed to use the data to provide services for which we’ve contracted and based on our direction. They are not allowed to use Personal Information for any other reason without the user’s consent or at the user’s direction.  You can find a list of third-party service providers we use [here](https://docs.google.com/spreadsheets/d/e/2PACX-1vQ_AYYT_4K9Hpc5LrL4QAeXUmEMQR7rAXrHtloM4yt3xNndqPh-ABHXy0SJHZQ8ZSDSFQQv7ZWdXQjj/pubhtml?gid=222345124&single=true). 
```

Comment:
It looks like the URL can have everything after `pubhtml` removed?

Reply:
thanks, done!